src/main/java/core/basesyntax/FIgureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FIgureSupplier {
```

Comment:
The class 'FIgureSupplier' should not begin with an empty line inside the class definition.